@xen-orchestra/lite/src/views/settings/SettingsView.vue

```diff
@@ -49,20 +49,37 @@
         </UiKeyValueRow>
       </UiKeyValueList>
     </UiCard>
-    <UiCard class="group">
-      <UiCardTitle>{{ $t("display") }}</UiCardTitle>
-      <UiKeyValueList>
-        <UiKeyValueRow>
-          <template #key>{{ $t("appearance") }}</template>
-          <template #value>
-            <FormSelect v-model="colorMode">
-              <option value="auto">{{ $t("theme-auto") }}</option>
-              <option value="dark">{{ $t("theme-dark") }}</option>
-              <option value="light">{{ $t("theme-light") }}</option>
-            </FormSelect>
-          </template>
-        </UiKeyValueRow>
-      </UiKeyValueList>
+    <UiCard class="group appearance">
+      <UiCardTitle>{{ $t("appearance") }}</UiCardTitle>
+      <div class="options">
+        <div
+          v-for="colorModeOption in colorModeOptions"
+          :key="colorModeOption"
+          class="option"
+          :class="{ selected: colorMode === colorModeOption }"
+          @click="colorMode = colorModeOption"
+        >
+          <img
+            v-if="colorModeOption === 'light'"
+            src="@/assets/color-mode-light.svg"
+            alt="Color mode light"
+          />
+          <img
+            v-else-if="colorModeOption === 'dark'"
+            src="@/assets/color-mode-dark.svg"
+            alt="Color mode dark"
+          />
+          <img
+            v-else
+            src="@/assets/color-mode-auto.svg"
+            alt="Color mode auto"
+          />
```

Comment:
```suggestion
          <img
            :alt="`Color mode ${colorModeOption}`"
            :src="`/src/assets/color-mode-${colorModeOption}.svg`"
          />
```

Reply:
It is not possible to use this kind of absolute path.

When built and used in production, these paths will be used as is (`/src/assets/...`) and will not be found.

Reply:
Also, the `@` resolution doesn't work well with dynamic URLs, that's why I split it in 3 `img`s.

Reply:
Yes. One way would be to import images and create an object like the following example, but I don't think there's any real point.

```ts
import colorModeLight from "@/assets/color-mode-light.svg";
import colorModeDark from "@/assets/color-mode-dark.svg";
import colorModeAuto from "@/assets/color-mode-auto.svg";

const colorModes = {
  light: colorModeLight,
  dark: colorModeDark,
  auto: colorModeAuto,
};
```

Then in template:

```vue
<img :src="colorModes[colorModeOption]" :alt="..." />
```

Reply:
That's kind of what I did at first but IMO it's too much indirection for only 3 `img`s so I suggest we keep it like that for this one.